requirements.txt

```
@@ -12,7 +12,7 @@ dm-sonnet # used for GNNs
 # TMVA: PyMVA interfaces
 scikit-learn
 tensorflow<2.16 ; python_version < "3.12"
-torch
+torch<2.5
```

Can this work?
```suggestion
torch!=2.5.*
```

We wouldn't be stuck in the past if 2.6 is available.

---

Yes, I will monitor the releases and check when they fix that bug.